AmplifyPlugins/DataStore/AWSDataStoreCategoryPluginIntegrationTests/AWSDataStorePluginConfigurationTests.swift

```diff
@@ -0,0 +1,28 @@
+//
+// Copyright 2018-2020 Amazon.com,
+// Inc. or its affiliates. All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import XCTest
+import Amplify
+import AWSDataStoreCategoryPlugin
+
+class AWSDataStorePluginConfigurationTests: XCTestCase {
+
+    // Note this test requires the ability to write a new database in the Documents directcory, so it must be embedded
+    // in a host app
+    func testDoesNotThrowOnMissingConfig() throws {
```

Comment:
unrelated to this PR, leaving a note for @DongQuanRui this looks like the right place to put tests for the configuring DataStore when schema version changes